wastewater/dataModel.WasteWaterTreatment/WasteWaterTreatment-schema.json

```diff
@@ -0,0 +1,760 @@
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "enum": [
+                        "Property"
+                    ]
+                },
+                "value": {
+                    "type": "number"
+                },
+                "unitCode": {
+                    "type": "string"
+                },
+                "observedAt": {
+                    "type": "string",
+                    "format": "date-time"
+                }
+            },
+            "required": [
+                "type",
+                "value"
+            ]
+        },
+        "NO3": {
+            "type": "object",
+            "description": "Property. Nitrate concentration measured in a tank",
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "enum": [
+                        "Property"
+                    ]
+                },
+                "value": {
+                    "type": "number"
+                },
+                "unitCode": {
+                    "type": "string"
+                },
+                "observedAt": {
+                    "type": "string",
+                    "format": "date-time"
+                }
+            },
+            "required": [
+                "type",
+                "value"
+            ]
+        },
+        "DO": {
+            "type": "object",
+            "description": "Property. Dissolved oxygen concentration measured in a tank",
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "enum": [
+                        "Property"
+                    ]
+                },
+                "value": {
+                    "type": "number"
+                },
+                "unitCode": {
+                    "type": "string"
+                },
+                "observedAt": {
+                    "type": "string",
+                    "format": "date-time"
+                }
+            },
+            "required": [
+                "type",
+                "value"
+            ]
+        },
+        "Redox": {
+            "type": "object",
+            "description": "Property. Redox level measured in a tank",
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "enum": [
+                        "Property"
+                    ]
+                },
+                "value": {
+                    "type": "number"
+                },
+                "unitCode": {
+                    "type": "string"
+                },
+                "observedAt": {
+                    "type": "string",
+                    "format": "date-time"
+                }
+            },
+            "required": [
+                "type",
+                "value"
+            ]
+        },
+        "Sludge level": {

```

Comment:
Subproperties defined in the NGSI LD standard are not necessary (i.e. unitcode, observedAt, etc) it should be removed